src/Firebase/Contract/Auth.php

```diff
@@ -47,7 +47,7 @@ interface Auth
     public function getUser($uid): UserRecord;
 
     /**
-     * @param array<Stringable|string> $uids
+     * @param non-empty-list<Stringable|string> $uids
```

Comment:
You might not care (I don't usually vendor tag), but this one is unique to PHPStan:
```suggestion
     * @param array<Stringable|string> $uids
     * @phpstan-param non-empty-list<Stringable|string> $uids
```

Reply:
I never really thought about it and assumed PHPStan and Psalm supported what the other provides 🙈. Thank you for telling me, I think I will then just use `array` instead of `list`, I think vendor-prefixing would add to much overhead (for me 😅)

Reply:
Yeah that's where I am at with my own stuff. We had this conversation recently at CodeIgniter because we want to support a variety of tools people use downstream, especially IDEs. So we are using vendor tags. 

I think Psalm supports `non-empty-list` already but PHPStorm does not? It's been a bit since I looked. What I really want is a big spreadsheet of all the annotations and which tools support them 🙄

Reply:
I'm using the PHPStan integration in PHPStorm and disabled the standard PHPStorm inspections that conflict with them.

Thinking about it, I think I will still keep the PHPStan-known annotations - I'm sure PHPStan and Psalm eventually will keep up with each other, and if they don't I hope people using Psalm will notify me and then I can add them as vendor-prefixed annotations 🤔

I might of course change my mind again, as I often do 😄

Reply:
This is basically where I am at with my own OSS stuff, so no complaints here! 🤷‍♂️